delta/rdf/src/test/scala/ch/epfl/bluebrain/nexus/delta/rdf/Fixtures.scala

```diff
@@ -21,14 +20,14 @@ trait Fixtures
   val iri = iri"http://nexus.example.com/john-doé"
 
   // format: off
-  val remoteContexts: Map[Iri, Json] =
+  val remoteContexts: Map[Iri, ContextValue] =
     Map(
-      iri"http://example.com/cöntéxt/0"  -> json"""{"@context": {"deprecated": {"@id": "http://schema.org/deprecated", "@type": "http://www.w3.org/2001/XMLSchema#boolean"} }}""",
-      iri"http://example.com/cöntéxt/1"  -> json"""{"@context": ["http://example.com/cöntéxt/11", "http://example.com/cöntéxt/12"] }""",
-      iri"http://example.com/cöntéxt/11" -> json"""{"@context": {"birthDate": "http://schema.org/birthDate"} }""",
-      iri"http://example.com/cöntéxt/12" -> json"""{"@context": {"Other": "http://schema.org/Other"} }""",
-      iri"http://example.com/cöntéxt/2"  -> json"""{"@context": {"integerAlias": "http://www.w3.org/2001/XMLSchema#integer", "type": "@type"} }""",
-      iri"http://example.com/cöntéxt/3"  -> json"""{"@context": {"customid": {"@type": "@id"} } }"""
+      iri"http://example.com/cöntéxt/0"  -> json"""{"@context": {"deprecated": {"@id": "http://schema.org/deprecated", "@type": "http://www.w3.org/2001/XMLSchema#boolean"} }}""".topContextValueOrEmpty,
```

Comment:
I wonder if:
```scala
ContextValue(json"""{"deprecated": {"@id": "http://schema.org/deprecated", "@type": "http://www.w3.org/2001/XMLSchema#boolean"} }""")
```
would not be more lisible ?

Reply:
This is just fixtures on tests, both are quite the same in terms or readability.